src/js/utils.js

```diff
@@ -0,0 +1,33 @@
+// Shared utility functions
+
+"use strict";
+
+/*
+ * Convert a date into the format of "Month day, Year".
+ * @param {String} inputDate - A string of date.
+ * @return The string in the format of "Month day, Year".
+ */
+// eslint-disable-next-line
+const convertDate = function (inputDate) {
+    const dateObject = new Date(inputDate);
+
+    const months = ["January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"];
+
+    return `${months[dateObject.getMonth()]} ${dateObject.getDate()}, ${dateObject.getFullYear()}`;
+};
+
+/*
+ * Replace special characters in the input string to understandable characters.
```

Comment:
Some people might question whether characters with accents are "not understandable". Was this code imported from somewhere? Could it be sourced from a library or some well-known algorithm?

Reply:
The installation of 11ty comes with a "slugify" node module to be used in node.js scripts. This function was created to be used directly in browsers. As now we don't yet have this requirement, I've removed this function.